source/docs/software/wpilib-tools/axon/data-format.rst

```diff
@@ -0,0 +1,35 @@
+:orphan:
+
+Data Format
+===========
+
+Accepted formats:
+- Supervisely
+- TensorFlow Object Detection CSV
+
+Supervisely
+-----------
+
+..
+   Taken from https://raw.githubusercontent.com/wpilibsuite/frc-docs/master/source/docs/software/examples-tutorials/machine-learning/setting-up-the-data.rst for temperary use
+
+1. `Supervisely <https://supervise.ly/>`__ is the web tool we use for labeling our data. Create a supervise.ly account to access our dataset. On the supervise.ly website, the Signup box is in the top right corner. Provide the necessary details, then click "CREATE AN ACCOUNT".
+2. Create a Workspace for your team. In your workspace, by click 'Members' on the left and then 'INVITE' at the top. After typing in your teammates' details, they should be able to view the workspace.
+3. To attain the WPILib dataset, `click this link <https://app.supervise.ly/share-links/zU1hctCmBs4rkglGXRzsmh5GbeAeqQ50ZUsGxtI9JNNR2SSbTnbMHvOiyeUgYw10>`__. After choosing which workspace you want to use, click clone.
```

Comment:
No constant links. They can download images off of the FRC kickoff page, and label them themselves. We can publish datasets in the blog.